lib/clients.ex

```diff
@@ -0,0 +1,44 @@
+defmodule GlassFactoryApi.ClientNotFound do
+  defexception message: "Client not found!"
+end
+
+defmodule GlassFactoryApi.Clients do
+  @moduledoc """
+  Provides the methods to list clients of an organization.
```

Comment:
Question: What would be client? It's like Plataformatec as a company or would be an employee from Plataformatec?

Reply:
In the context we use it here in Plataformatec, is a company which hires Ptec's services.

But the context could be different for other companies using GlassFactory.